lib/msal-core/src/telemetry/TelemetryTypes.ts

```diff
@@ -0,0 +1,20 @@
+import TelemetryEvent from "./TelemetryEvent";
+
+export interface InProgressEvents { [key: string] : TelemetryEvent; }
+export interface EventCount { [eventName: string] : number; }
+export interface EventCountByCorrelationId { [correlationId: string] : EventCount; }
+export interface CompletedEvents { [correlationId: string ] : Array<TelemetryEvent>; }
+
+// SDK SHOULD BE DEFAULTED and Pulled from Package
+export interface TelemetryPlatform {
+    sdk: string;
```

Comment:
Is this to record whether we are using angular, react, etc.?

Reply:
no this is for sdk: 'javascript' or msaljs. To differentiate between other msals  in telemetry aggregates

Reply:
Shouldn't we be setting this as a readonly then? Are we allowing devs to modify this value?

Reply:
yeah that is probably better

Reply:
I actually think I am going to wait till I get to the point where this is wired through to Configuration to see where the default should be.  TelemetryManager will be constructed by UAA so I just want to make sure I put the defaults at the right level.  Again this code should be able to be reused for any js platform lib we build

Reply:
No problem, just want to make sure we don't leave any room for the developer to mess up the log format we expect

Reply:
setting default isn't what I'm worried about here, its more of can the developer change the value and possibly change the format of expected logs.

Reply:
Yep, make sense.  I think that will happen in the config object based on what we expose right?

```js
new UserAgenApplication({
   system: { 
      telemetry: {  // dont expose sdk here so that we can configure it internally. Dev has no access
          telemetryCallback,
          applicationName,
          applicationVersion,
          onlySendFailureTelemetry?
      }
   }
})
```